### PR TITLE
[12.0] crm_deduplicate_filter: Migration to 12.0

### PR DESCRIPTION
**Pull request by victor-ditay**
Depends on:

- [ ] crm_deduplicate_acl

cc @Tecnativa

### CODE REVIEW

File: crm_deduplicate_filter/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* Pedro M. Baeza <pedro.baeza@tecnativa.com>
+* Vicent Cubells <vicent.cubells@tecnativa.com>
+* Luis M. Ontalba <luis.martinez@tecnativa.com>
```
Comment (pedrobaeza) [inline crm_deduplicate_filter/readme/CONTRIBUTORS.rst:3]: Include yourself here

File: crm_deduplicate_filter/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
+To use this module, you need to:
+
+#. Go to *CRM/Sales > Tools > Deduplicate Contacts*.
+#. Mark "'Is a company?' field selected", "Parent company not set" or
+   "Parent company set (Contacts)" in the section 'Exclude contacts having'.
+#. This criteria will be used for excluding in the deduplication the selected
+   kind of records.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```
Comment (pedrobaeza) [inline crm_deduplicate_filter/readme/USAGE.rst]: Don't include this part, as this is old stuff that now it's auto-generated in the header.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Next time is better to start with the lower dependencies module (`crm_deduplicate_acl`). Check how I have modified main comment for letting clear the dependency.

**Comment by victor-ditay** [comment]
@pedrobaeza I did the changes requested

**Comment by pedrobaeza** [comment]
This should be moved and renamed the same as base module, but better to wait for the other to be ready.

**Pull request closed without merging**